src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -39,12 +39,13 @@
     /**
      * Loads one or several .env files.
      *
-     * @param ...string A list of files to load
+     * @param string   $path  A file to load
+     * @param string[] $paths A list of additionnal files to load
```

Comment:
looking at eg http://stackoverflow.com/questions/14513356/phpdoc-documenting-a-function-with-a-variable-number-of-arguments
`...string` should be way to go, sorry for the bad comment